### ISSUE TITLE
Grandpa benchmarks do not match Grandpa::WeightInfo Trait

### ISSUE THREAD

**Issue by krisbitney** [paritytech/polkadot-sdk#3973]
### Is there an existing issue?

- [X] I have searched the existing issues

### Experiencing problems? Have you tried our Stack Exchange first?

- [X] This is not a support question.

### Description of bug

The Grandpa benchmarks do not match Grandpa::WeightInfo Trait, so it is not possible to automatically generate weights using the CLI.

Trait: https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/grandpa/src/lib.rs#L379
Benchmarks: https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/grandpa/src/benchmarking.rs

### Steps to reproduce

.

**Comment by ggwpez** [comment]
Yes this is a known issue. But no capacity to fix. https://github.com/paritytech/polkadot-sdk/issues/395
